example/errors/example.dart

```diff
@@ -107,8 +110,10 @@ void main() {
   });
 
   ErrorCreator errorCreator;
-  ButtonElement createErrorCreatorButton = querySelector('#create-ErrorCreator-button');
-  ButtonElement disposeErrorCreatorButton = querySelector('#dispose-ErrorCreator-button');
+  ButtonElement createErrorCreatorButton =
+      querySelector('#create-ErrorCreator-button');
+  ButtonElement disposeErrorCreatorButton =
+      querySelector('#dispose-ErrorCreator-button');
```

Comment:
This is just the formatter doing its thing. Someone must have forgotten to format? Or used a different version?